### PR TITLE
Fixed the timezome CSS bug

### LINKED ISSUE #1583: The buttons and links shift downwards when the "select timezone" input field is selected
**Actual Behaviour**

The right part of the page gets dislocated and the buttons and links shift downwards when the input field of select timezone is selected in the account option of settings.

**Expected Behaviour**

The page shouldn't get dislocated when the option is selected.

**Steps to reproduce it**

Go to settings/account. Try to select a timezone.

**Screenshots of the issue**

**Would you like to work on the issue?**

 Yes.

### PR DESCRIPTION
**Pull request by plxity**
Fixes #1583 

Changes: On using timezone dropdown the button and link don't shift downwards

Demo Link: https://pr-1911-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by plxity** [comment]
@akshatnitd Please review this.

File: src/components/ChatApp/Settings/Settings.css [hunk 1]
```diff
@@ -175,6 +175,13 @@
 .email-field:hover{
     cursor: not-allowed;
 }
+.time-zone{
+  padding-bottom: 30px
```
Comment (AakashMallik) [inline src/components/ChatApp/Settings/Settings.css]: use multiple of 4 or 8

File: src/components/ChatApp/Settings/AccountTab.react.js [hunk 2]
```diff
@@ -73,14 +73,19 @@ const AccountTab = props => {
...
       <br />
-      <TimezonePicker
-        value={props.timeZone}
-        onChange={timezone => props.handleTimeZone(timezone)}
-        inputProps={{
-          placeholder: 'Select Timezone...',
-          name: 'timezone',
-        }}
-      />
+
```
Comment (AakashMallik) [inline src/components/ChatApp/Settings/AccountTab.react.js]: New line not needed

File: src/components/ChatApp/Settings/AccountTab.react.js [hunk 3]
```diff
@@ -73,14 +73,19 @@ const AccountTab = props => {
         <Translate text="Select TimeZone" />
       </div>
       <br />
```
Comment (AakashMallik) [inline src/components/ChatApp/Settings/AccountTab.react.js]: <br/> not needed, instead use margin in your time-zone class

File: src/components/ChatApp/Settings/Settings.css (hunk 1 above)
Comment (plxity) [reply src/components/ChatApp/Settings/Settings.css]: Why is it so?

File: src/components/ChatApp/Settings/AccountTab.react.js (hunk 3 above)
Comment (plxity) [reply src/components/ChatApp/Settings/AccountTab.react.js]: I did not get you.

File: src/components/ChatApp/Settings/Settings.css (hunk 1 above)
Comment (AakashMallik) [reply src/components/ChatApp/Settings/Settings.css]: Material standards.

File: src/components/ChatApp/Settings/AccountTab.react.js (hunk 3 above)
Comment (AakashMallik) [reply src/components/ChatApp/Settings/AccountTab.react.js]: \<br/> tag is not needed. Instead, use margin. The br tag got parsed as html in my comment.

File: src/components/ChatApp/Settings/Settings.css (hunk 1 above)
Comment (sakshee-19) [reply src/components/ChatApp/Settings/Settings.css]: Add semicolon too.

**Comment by plxity** [comment]
@akshatnitd This PR is same as in accounts.susi.ai. Please review this.

**Comment by plxity** [comment]
@akshatnitd @anshumanv  Please review this. It is the same PR as in account.susi.ai. I squashed my commits 5 to 3. Due to some issue in my terminal, I am not to able to do it more. Can you squash and merge?

**Comment by fragm3** [comment]
@pixity try: https://davidwalsh.name/squash-commits-git

**Comment by plxity** [comment]
@fragm3 Okay, Thank you. I will try.

**Pull request closed without merging**